### PR TITLE
Prepare v1.17 stable branch

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

The CI has passed, the Documentation failed, the new change is [here](https://github.com/cilium/cilium/compare/e1e39a835147fc1a72a234c4d694a0a59af9d197..7df8e0848ff13c42158c9d343540b7d9fb8e8eea). The successful test run is [here](https://github.com/cilium/cilium/actions/runs/12394585424?pr=36627)

### CODE REVIEW

File: .github/workflows/fqdn-perf.yaml
Comment (joestringer) [inline .github/workflows/fqdn-perf.yaml:1]: Observing that this workflow is removed, but the ariane trigger is still configured.

```
$ git grep fqdn-perf.yaml
.github/ariane-config.yaml:    - fqdn-perf.yaml
```

Comment (joestringer) [reply .github/workflows/fqdn-perf.yaml:1]: Same for `net-perf-gke.yaml`

File: .github/workflows/tests-cifuzz.yaml
Comment (joestringer) [inline .github/workflows/tests-cifuzz.yaml:1]: This reminds me. I recently re-enabled this on `main` but it's also running on older stable branches. It probably makes sense to remove it on the older branches too, but we can't do that from this PR.

File: stable.txt
Comment (joestringer) [inline stable.txt:1]: I see that f5bdf28e8af1b44f719aac4e1524a5212d9424a3 (v1.16 prep) also changed the `VERSION`, not sure whether we need to do that here or defer that to a subsequent commit.

File: .github/workflows/build-images-ci.yaml [hunk 1]
```diff
@@ -112,8 +112,12 @@ jobs:
...
             tag=${{ github.sha }}
           fi
-          if [[ "${{ github.event_name == 'push' }}" == "true" && "${{ github.ref_name }}" == "${{ github.event.repository.default_branch }}" ]]; then
-            floating_tag=latest
+          if [[ "${{ github.event_name == 'push' }}" == "true" ]]; then
+            if [[ "${{ github.ref_name }}" == "${{ github.event.repository.default_branch }}" ]]; then
+              floating_tag=latest
+            else
+              floating_tag="${{ github.ref_name }}"
+            fi
```
Comment (joestringer) [inline .github/workflows/build-images-ci.yaml]: Why is this necessary this time?

File: .github/workflows/tests-datapath-verifier.yaml [hunk 2]
```diff
@@ -108,7 +105,7 @@ jobs:
           - kernel: '6.12-20241214.023514'
             ci-kernel: '61'
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
-          - kernel: 'bpf-next-20241213.013358'
+          - kernel: '6.12-20241216.084703'
```
Comment (julianwiedmann) [inline .github/workflows/tests-datapath-verifier.yaml]: Note that we already test with kernel 6.12 (one entry up). So I think this can just go away.

File: .github/workflows/tests-cifuzz.yaml
Comment (joestringer) [reply .github/workflows/tests-cifuzz.yaml:1]: https://github.com/cilium/cilium/pull/36641
https://github.com/cilium/cilium/pull/36642
https://github.com/cilium/cilium/pull/36643

File: stable.txt
Comment (aanm) [reply stable.txt:1]: Let's defer it to a subsequente commit because I believe this will be modified by the release tool.

**Review by julianwiedmann — APPROVED** [review]

**Comment by aanm** [comment]
/test-backport-1.17

**Comment by aanm** [comment]
The CI has passed, the Documentation failed, the new change is [here](https://github.com/cilium/cilium/compare/e1e39a835147fc1a72a234c4d694a0a59af9d197..7df8e0848ff13c42158c9d343540b7d9fb8e8eea). The successful test run is [here](https://github.com/cilium/cilium/actions/runs/12394585424?pr=36627)

**Review by rgo3 — APPROVED** [review]

**Review by joestringer — APPROVED** [review]
